exo-go/test_helpers/tutorial_feature_context.go

```diff
@@ -0,0 +1,77 @@
+package testHelpers
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"path"
+	"strings"
+
+	"github.com/DATA-DOG/godog"
+	"github.com/DATA-DOG/godog/gherkin"
+	"github.com/Originate/exosphere/exo-go/src/util"
+	"github.com/jaytaylor/html2text"
+)
+
+// TutorialFeatureContext defines the festure context for the tutorial
+// nolint gocyclo
+func TutorialFeatureContext(s *godog.Suite) {
+
+	s.Step(`^I see "([^"]*)" in the terminal$`, func(expectedText string) error {
+		return process.WaitForTextWithTimeout(expectedText, 1000)
+	})
+
+	s.Step(`^I am in an empty folder$`, func() error {
+		return nil
+	})
+
+	s.Step(`^I cd into "([^"]*)"$`, func(dir string) error {
+		appDir = path.Join("tmp", dir)
+		return nil
+	})
+
+	s.Step(`^my application contains the template folder "([^"]*)"$`, func(templateDir string) error {
+		templateName := strings.Split(templateDir, "/")[1]
+		return checkoutServiceTemplate(appDir, templateName)
+	})
+
+	s.Step(`^waiting until I see "([^"]*)" in the terminal$`, func(expectedText string) error {
+		return process.WaitForTextWithTimeout(expectedText, 60000)
+	})
+
+	s.Step(`^I stop all running processes$`, func() error {
+		if process != nil {
+			return process.Kill()
+		}
+		return nil
+	})
+
+	s.Step(`^the file "([^"]*)":$`, func(filePath string, expectedContent *gherkin.DocString) error {
+		return ioutil.WriteFile(path.Join(appDir, filePath), []byte(expectedContent.Content), 0777)
+	})
+
+	s.Step(`^http:\/\/localhost:(\d+) displays:$`, func(port string, expectedContent *gherkin.DocString) error {
+		resp, err := http.Get(fmt.Sprintf("http://localhost:%s", port))
+		if err != nil {
+			return err
+		}
+		defer resp.Body.Close()
+		body, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			return err
+		}
+		formattedText, err := html2text.FromString(string(body))
+		if err != nil {
+			return err
+		}
+		text := util.Strip(`\*+\n|\-+\n`, formattedText)
```

Comment:
This strips out the `*********` and `----------` lines that `hmtl2text` adds to the text https://github.com/jaytaylor/html2text#example-usage